### PR TITLE
Scroll snap: see also sections

### PR DESCRIPTION
**Pull request by estelle**

### CODE REVIEW

File: files/en-us/web/css/scroll-snap-type-x/index.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ The **`scroll-snap-type-x`** [CSS](/en-US/docs/Web/CSS) property defines how str
 
 Specifying any precise animations or physics used to enforce those snap points is not covered by this property but instead left up to the user agent.
 
+> **Note:** The CSS [scroll-snap-type](/en-US/docs/Web/CSS/scroll-snap-type) is the modern specification that defines this functionality.
+
```
Comment (estelle) [inline files/en-us/web/css/scroll-snap-type-x/index.md:24]:
```suggestion
```

File: files/en-us/glossary/snap_positions/index.md [hunk 2]
```diff
@@ -7,4 +7,4 @@ tags:
 
 A [scroll container](/en-US/docs/Glossary/Scroll_container) may set **snap positions** — points that the [scrollport](/en-US/docs/Glossary/Scrollport) will stop moving at after a scrolling operation is completed. This allows a scrolling experience that gives the effect of paging through content rather than needing to drag content into view.
 
-Defining Snap positions on the scroll container was introduced in the [CSS Scroll Snap specification](/en-US/docs/Web/CSS/CSS_Scroll_Snap).
+Defining snap positions on the scroll container was introduced in the [CSS Scroll Snap specification](/en-US/docs/Web/CSS/CSS_Scroll_Snap).
```
Comment (dipikabh) [inline files/en-us/glossary/snap_positions/index.md:10]:
```suggestion

```
Added this to previous para

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks, @estelle! Suggested a few edits.

File: files/en-us/web/css/scroll-snap-type-y/index.md [hunk 3]
```diff
@@ -18,6 +18,8 @@ browser-compat: css.properties.scroll-snap-type-y
 
 The **`scroll-snap-type-y`** [CSS](/en-US/docs/Web/CSS) property defines how strictly snap points are enforced on the vertical axis of the scroll container in case there is one.
 
+> **Note:** The CSS [scroll-snap-type](/en-US/docs/Web/CSS/scroll-snap-type) is the modern specification that defines this functionality.
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/scroll-snap-type-y/index.md]: I prefer to leave it where it is. there are two large red boxes above line 19. I think that area is too busy, and the note will be more likely to be read if it is in the main reading area.

**Review by dipikabh — APPROVED** [review]
Looks good. Thanks for the updates, @estelle!

**Comment by dipikabh** [comment]
Markdownlint is complaining about:
"Warning: files/en-us/glossary/snap_positions/index.md:12 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]

**Pull request merged**